packages/design-system/src/components/Form/Affix/AffixStyles.module.scss

```diff
@@ -37,14 +37,13 @@
 	min-height: auto;
 	max-height: tokens.$coral-sizing-m;
 	transition: tokens.$coral-transition-fast;
+	border-radius: 2px 0 0 2px;
+	border-right: tokens.$coral-border-s-solid tokens.$coral-color-neutral-border-weak;
 
-	&:first-child {
-		border-radius: 2px 0 0 2px;
-		border-right: tokens.$coral-border-s-solid tokens.$coral-color-neutral-border-weak;
-	}
 
-	&:last-child {
+	&_isSuffix {
```

Comment:
Browser plugins love to push shit elements in form inputs (looking at you Dashlane) so :first-of / :last-of stuff won't work.